dev-lang/julia-bin/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
Please add
```
   <maintainer type="person">                                                     
     <email>tamiko@gentoo.org</email>                                             
     <name>Matthias Maier</name>                                                  
   </maintainer>
```
on top. As primary maintainer of julia I would like to maintain this package as well.